### PR TITLE
Form styles

### PR DESCRIPTION
**Pull request by dreuss**
Adds a default BG color to the form field, along with default text color.  The widget also has the capability to over-ride these defaults and put the control into the builders hands for the color over-ride options.

### CODE REVIEW

File: public/static/components/self-storage-iui-cards/show.html [hunk 1]
```diff
@@ -35,18 +35,25 @@
     .self-storage-iui-cards .iui-cards-wrapper .iui-cards-unit-description,
     .self-storage-iui-cards .iui-cards-wrapper .iui-cards-unit-actions a.is-phone,
     .self-storage-iui-cards .iui-cards-wrapper .iui-cards-unit-actions a.is-phone:hover,
-    .self-storage-iui-cards .iui-cards-wrapper .iui-cards-unit-actions a.is-phone span,
+    .self-storage-iui-cards .iui-cards-wrapper .iui-cards-unit-actions a.is-phone span{ color:{{ widget.primary_color.best_value }}; }
+    .self-storage-iui-cards .iui-cards-wrapper a.iui-cards-nearby-location-card div.iui-cards-nearby-distance { border-top:1px solid {{ widget.primary_color.best_value }}; }
+    .self-storage-iui-cards .iui-cards-wrapper .iui-cards-category-card,
+    .self-storage-iui-cards .iui-cards-wrapper .iui-cards-unit-header,
+    .self-storage-iui-cards .iui-cards-wrapper .iui-cards-nearby-locations.nearby-locations-open { background:{{ widget.primary_color.best_value }}; }
```
Comment (crissmancd) [inline public/static/components/self-storage-iui-cards/show.html]: Please use background-color for this

File: public/static/components/self-storage-iui-cards/show.html [hunk 2]
```diff
@@ -35,18 +35,25 @@
...
+
+    {% if widget.form_bg_color.value != blank %}
+    .self-storage-iui-cards .iui-cards-wrapper .leads-form{ background:{{ widget.form_bg_color.value }}; }
```
Comment (crissmancd) [inline public/static/components/self-storage-iui-cards/show.html]: and also background-color for this

**Comment by crissmancd** [comment]
I'm thinking that using .value instead of .best_value might be better, since each location probably uses their own custom colors. We don't want it to by default inherit another location's colors. Just a suggestion. If there's a reason you're using best_value then go for it.

Approved after changing the css property.

**Comment by dreuss** [comment]
@crissmancd I ended up just re-arranging some of the pre-existing code, which happened to use the best_value stuff, so it happened to look like new code.  I'm not sure of the implications on removing that.  If you've had reliable success removing it before, I'm game.  Just don't want bork any existing implementations.  Thanks!

**Comment by crissmancd** [comment]
Then just keep it for now. Don't want to take any risks at the moment :)

**Pull request merged**